scrapers/BigBootyTGirls.yml

```diff
@@ -0,0 +1,27 @@
+name: BigBootyTGirls
+sceneByURL:
+  - action: scrapeXPath
+    url:
```

Comment:
```yaml
    url:
      - bigbootytgirls.com
```
covers all cases, no need to add the other 2

Reply:
Yup, forgot to remove those.